tests/secrets/test_cli.py

```diff
@@ -0,0 +1,162 @@
+# Databricks CLI
```

Comment:
A couple endpoints aren't tested in this file, namely:
deleteAcl
writeAcl
deleteSecret
deleteScope

Can we add some simple tests for these?